test/Microsoft.ML.Predictor.Tests/TestAutoInference.cs

```diff
@@ -201,206 +202,209 @@ public void EntryPointPipelineSweep()
+            Assert.Equal(allPipelines.Length, numIterations);
+            Assert.True(bestPipeline.PerformanceSummary.MetricValue > 0.1);
 
-            //var results = runner.GetOutput<IDataView>("ResultsOut");
-            //Assert.IsNotNull(results);
-            //var rows = PipelinePattern.ExtractResults(Env, results, "Graph", "MetricValue", "PipelineId");
-            //Assert.IsTrue(rows.Length == numIterations);
+            var results = runner.GetOutput<IDataView>("ResultsOut");
+            Assert.NotNull(results);
+            var rows = PipelinePattern.ExtractResults(Env, results,
+                "Graph", "MetricValue", "PipelineId", "TrainingMetricValue", "FirstInput", "PredictorModel");
+            Assert.True(rows.Length == numIterations);
         }
 
         [Fact(Skip = "Need CoreTLC specific baseline update")]
         public void TestTextDatasetLearn()
         {
             //using (var env = new TlcEnvironment())
             //{
-                //string pathData = GetDataPath(@"../UnitTest/tweets_labeled_10k_test_validation.tsv");
-                //int batchSize = 5;
-                //int numIterations = 35;
-                //int numTransformLevels = 1;
-                //int numSampleRows = 100;
-                //AutoInference.SupportedMetric metric = AutoInference.SupportedMetric.AccuracyMicro;
-
-                //// Using the simple, uniform random sampling (with replacement) engine
-                //PipelineOptimizerBase autoMlEngine = new UniformRandomEngine(env);
-
-                //// Test initial learning
-                //var amls = AutoInference.InferPipelines(env, autoMlEngine, pathData, "", out var _, numTransformLevels, batchSize,
-                    //metric, out var _, numSampleRows, new IterationTerminator(numIterations),
-                    //MacroUtils.TrainerKinds.SignatureMultiClassClassifierTrainer);
-                //env.Check(amls.GetAllEvaluatedPipelines().Length == numIterations);
+            //string pathData = GetDataPath(@"../UnitTest/tweets_labeled_10k_test_validation.tsv");
+            //int batchSize = 5;
+            //int numIterations = 35;
+            //int numTransformLevels = 1;
+            //int numSampleRows = 100;
+            //AutoInference.SupportedMetric metric = AutoInference.SupportedMetric.AccuracyMicro;
+
+            //// Using the simple, uniform random sampling (with replacement) engine
+            //PipelineOptimizerBase autoMlEngine = new UniformRandomEngine(env);
+
+            //// Test initial learning
+            //var amls = AutoInference.InferPipelines(env, autoMlEngine, pathData, "", out var _, numTransformLevels, batchSize,
+            //metric, out var _, numSampleRows, new IterationTerminator(numIterations),
+            //MacroUtils.TrainerKinds.SignatureMultiClassClassifierTrainer);
+            //env.Check(amls.GetAllEvaluatedPipelines().Length == numIterations);
             //}
             //Done();
         }
 
```

Comment:
should we just delete this test?

Reply:
or maybe uncomment content, they test system skip this test anyway

---
In reply to: [188105947](https://github.com/dotnet/machinelearning/pull/152#discussion_r188105947) [](ancestors = 188105947)